game-core/src/test/java/games/strategy/triplea/delegate/ExecutionStackTest.java

```diff
@@ -0,0 +1,87 @@
+package games.strategy.triplea.delegate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import games.strategy.engine.delegate.IDelegateBridge;
+
+class ExecutionStackTest {
+  private final ExecutionStack dummy = new ExecutionStack();
```

Comment:
Sorry, forgot to add this comment to my previous review...

Curious choice for a field name.  Could you elaborate on why you chose `dummy`?  When I see something named as such, I think of a [_Dummy Object_](https://en.wikipedia.org/wiki/Test_double), but this looks like it is simply the object under test.  In that case, `executionStack` seems more appropriate.